functions/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "incremental": true,
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "noEmit": true,
+    "strict": true,
+    "target": "ES6",
+    "tsBuildInfoFile": "../node_modules/.cache/tsbuildinfo/jest", // avoid clobbering the build tsbuildinfo
+    "types": ["jest", "node"],
+    "jsx": "react",
+    "moduleResolution": "NodeNext",
```

Comment:
```suggestion
    "jsx": "react"
```

Reply:
I only added this since for some reason without the setting I was getting an error with imports and package resolution. 

<img width="967" alt="Screenshot 2023-07-07 at 10 36 56 AM" src="https://github.com/Uniswap/interface/assets/35351983/e7bc514e-b8da-4f4e-9a5b-9589a083d63d">

Is there a better way to resolve this issue?

Reply:
Weird. I think you should leave as is, then. We can always review/polish later.